astropy/utils/console.py

```diff
@@ -691,6 +691,13 @@ def work(i):
             `sys.stdout`.  If `file` is not a tty (as determined by
             calling its `isatty` member, if any), the scrollbar will
             be completely silent.
+
+        step : int, optional
+            Update the progress bar at least every *step* steps (default: 100).
+            If ``multiprocess`` is `True`, this will affect the size
+            of the chunks of ``items`` that are submitted as separate tasks
+            to the process pool.  A large step size may make the job
+            complete faster if `items` is very long.
```

Comment:
@embray: I'm confused now, because the dev docs only seem to mention single back-ticks ( https://astropy.readthedocs.org/en/stable/development/docrules.html#common-rest-concepts ), and travis seems to fail over the double ones?